### PR TITLE
Update SRG-OS-000114-GPOS-00059 for RHEL 9 STIG

### PR DESCRIPTION
**Pull request by jan-cerny**
#### Description:

adds fix texts, Ansible remediations, fixes OCIL

#### Rationale:

RHEL 9 STIG

### CODE REVIEW

**Review by Mab879 — COMMENTED** [review]
Just a few comments. If you can provide a source for device IDs that would be helpful.

File: linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount_open/rule.yml [hunk 1]
```diff
@@ -56,3 +57,21 @@ ocil: |-
     If properly configured, the output for <tt>automount-open</tt> should be <tt>/org/gnome/desktop/media-handling/automount-open</tt>
 
 platform: machine
+
+fix: |-
+    Configure GNOME 3 to disable automated mount of removable media.
+
+    First, add or update the following section
```
[the remark this answers is not in the document]
Comment (Mab879) [reply linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount_open/rule.yml:64]: That sounds reasonable.

**Comment by jan-cerny** [comment]
I have rebased this pull request on the top of the currrent master branch and I have replaced fix by fixtext and resolved conflict with moving Jinja macro to a different file.

File: linux_os/guide/system/permissions/mounting/service_autofs_disabled/rule.yml [hunk 2]
```diff
@@ -60,9 +60,15 @@ references:
...
+
 platform: machine
 
 template:
     name: service_disabled
     vars:
         servicename: autofs
+
+fixtext: |-
+    Configure the operating system to disable the ability to automount devices.
```
Comment (Mab879) [inline linux_os/guide/system/permissions/mounting/service_autofs_disabled/rule.yml:73]:
```suggestion
    Configure {{{ full_name }}} to disable the ability to automount devices.
```

**Review by Mab879 — CHANGES_REQUESTED** [review]
Just one minor change requested.

Thanks for the PR.

**Review by Mab879 — APPROVED** [review]

**Comment by Mab879** [comment]
@jan-cerny Can you take look a the merge conflict?

**Comment by jan-cerny** [comment]
Hi, it's now rebased again.

**Comment by Mab879** [comment]
So those OpenShift CI errors are giving me some pause before I merged them.

@rhmdnd or @Vincent056 I think these failures might be valid, can you please confirm?

**Comment by jan-cerny** [comment]
I have resolved the conflict and I have rebased on the top of the latest upstream branch.

**Comment by Vincent056** [comment]
Since https://github.com/ComplianceAsCode/content/pull/8564 has just merged, the test should pass once this PR rebase.

**Comment by jan-cerny** [comment]
I have rebased on the top of the latest upstream branch.

**Comment by Mab879** [comment]
Waving SSGTS, the tests pass locally.

**Pull request merged**